### ISSUE TITLE
[Hunter][Spell] Feign death

### ISSUE THREAD

**Issue by Zarant** [Atlantiss/NetherwingBugtracker#4842]
**Description**: Mobs don't reset properly if a hunter with the feign death buff is still on their threat table

**Current behaviour**: Feign death only prevents mobs from attacking you if you're the last person remaining on their threat list. Incidental aoe damage that put you back in combat while feigning will make mobs ignore the feign death buff.
It's very easy to replicate, have your pet tank something, feign and then let your pet die, mobs will ignore feign death and aggro you.

**Expected behaviour**: Mobs should treat the hunters with the feign death buff the same way they treat dead players. 

Source: https://youtu.be/dXE_RR5XzE8?t=638 - Video from retail vanilla. Hunter feigns, pet dies after feign death and then the boss is reset. Also notice that the hunter is in combat during feign death and then he drops out of combat only after the pet died.

**Server Revision**: 2756

**Comment by ErothT** [comment]
Can confirm that feign death is bugged and is a death sentence at times.

**Comment by Kristoferhh** [comment]
Can confirm this is still bugged (pls fix). Another bug with FD is that it often doesn't show the cooldown of the spell, whether it's resisted or not.

**Comment by Argonn** [comment]
1. Feign dead sometimes breaks (cannot use again, saying its still on cooldown till relog)
2. Feign dead work wont, if you are not the last person alive. If a party member dies, after you feign everyone will come for you even when laying on the floor.
3. Feign dead hardly work if there is a enemy in melee range hitting you (50-75% resist)
4. An aura/aoe/heal will get your corpse right back into combat, making it useless.

For now, feign dead is just as usefull as 'fade' it drops agro, but don't use it for anything else.

**Comment by RoboboR77** [comment]
Feign death works as intended without a pet (your aggro drops and the mob resets if nothing else has threat on the mob). However if you feign death with a pet out, the mob will attack the pet (intended) and after the pet is dead will immediately go back to attacking your feign death body (bugged). 

If feign death is successfully cast, the mob should fully reset after your pet and whoever else has threat on the mob dies. The youtube link of retail wow posted by the OP shows this clearly. Please fix this as right now feign death only acts as a threat drop and not an actual aggro reset if your pet is out which takes out a huge amount of utility from feign death.

**Comment by Cimetta** [comment]
As others have said, it's not working as it should and did back in retail. Some of the issues that happen continuously:
1. The ability will sometimes "Grey out", making it unusable until the hunter restarts the game client completely,
2. Almost half the time the ability is used, Feign death won't say "resisted" or use the ability's cool-down, it just won't do anything,
3. When it does indeed work, the mob will still attempt to kill you unless you were the last one on the threat table when using the ability and everyone else on the thread table is dead.

As another user commented, it is practically a buggy fade that doesn't work very often. It has been like this since at least January 2019.

**Comment by shorgan1991** [comment]
does it put you at the complete bottom of the aggro table?

**Comment by Cimetta** [comment]
> does it put you at the complete bottom of the aggro table?

If it does work, it does.

**Comment by ErothT** [comment]
If you use feign death before EVERY OTHER PERSON is dead (including your pet), the mobs will all run at you after killing everyone else and kill you.

**Comment by Dragunovka** [comment]
up

**Comment by Argonn** [comment]
> does it put you at the complete bottom of the aggro table?

Yes it does, it puts you on the bottom instead of wiping you off the table.

**Comment by Dragunovka** [comment]
up, still broken

**Comment by Tereneckla** [comment]
fixed rev 3052
